cluster/ci/monitoring/mixins/grafana_dashboards/ghproxy.jsonnet

```diff
@@ -21,6 +21,39 @@ dashboard.new(
         schemaVersion=18,
         refresh='1m',
       )
+.addTemplate(
+  template.new(
+    'token_hash',
+    'prometheus',
+    'label_values(github_request_duration_count{job="ghproxy"}, token_hash)',
+    label='token_hash',
+    allValues='.*',
+    includeAll=true,
+    refresh='time',
+  )
+)
+.addTemplate(
```

Comment:
You could add a function to return a new template variable and have the input be the label to use

Reply:
Done.